lib/session.js

```diff
@@ -33,11 +33,11 @@ module.exports = class Session {
       }
     }
 
-    this.touch()
     if (!this.encryptedSessionId) {
       this.encryptedSessionId = cookieSigner.sign(this.sessionId)
     }
     this[originalHash] = this[hash]()
+    this.touch()
```

Comment:
Do we need .touch() when we merge #152 ? Or did I make a mistake in #152

Reply:
Ah crap good catch.

I didn't merge your code in, let me do that...